### PR TITLE
renamed app menu screen & created event dashboard screen

### PR DESCRIPTION
**Pull request by darrenb-ta**
- Renamed app menu screen file to match new naming convention
- Added Event Dashboard screen to cover Access elements & Sales elements
Note: PR does not include graph elements as env issue, will add in later PR. Also excluded bottom menu nav buttons as these will be added in a new helper file.

### CODE REVIEW

**Review by pirasanthan-jesugeevegan — APPROVED** [review]

File: test/screens/eg-access-event-dashboard.screen.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    this.EventDashboardSalesLabel =
+      '//android.view.ViewGroup[@content-desc="Sales_Stat_Layout"]/android.view.ViewGroup/android.widget.TextView[1]';
+    this.EventDashboardSalesValueText =
+      '//android.view.ViewGroup[@content-desc="Sales_Stat_Layout"]/android.view.ViewGroup/android.widget.TextView[3]';
+    this.EventDashboardSalesTotalValueText =
+      '//android.view.ViewGroup[@content-desc="Sales_Stat_Layout"]/android.view.ViewGroup/android.widget.TextView[5]';
+    this.EventDashboardSalesLeftLabel =
+      '//android.view.ViewGroup[@content-desc="Sales_Stat_Layout"]/android.view.ViewGroup/android.widget.TextView[2]';
```
Comment (Pelumi-Festicket) [inline test/screens/eg-access-event-dashboard.screen.js:32]: EventDashboardSalesLeftLabel and EventDashboardSalesTimelineLabel have the same XPath here

Comment (darrenb-ta) [reply test/screens/eg-access-event-dashboard.screen.js:32]: Very similar but I think still valid,
ValueText ends with [5] 
LegtLabel ends with [2] 
I'll check in the morning to be sure though.

Comment (pirasanthan-jesugeevegan) [reply test/screens/eg-access-event-dashboard.screen.js:32]: Yes its not the same as the index is different meaning that its a different Element

**Review by Pelumi-Festicket — APPROVED** [review]
looks good 👍🏾

**Pull request merged**